packages/block-library/src/cover/index.js

```diff
@@ -245,10 +318,10 @@ export const settings = {
 							className={ className }
 							labels={ {
 								title: label,
-								name: __( 'an image' ),
+								name: __( 'an image or a video' ),
```

Comment:
Should have a `translators:` note reading something like

```
translators: Fragment of the sentence: "Drag %s, upload a new one or select a file from your library."
```

Aside: the way we interpolate these sentence fragments in MediaPlaceholder is pretty bad i18n.